code/game/mecha/mecha.dm

```diff
@@ -4,6 +4,7 @@
 	name = "Mecha"
 	desc = "Exosuit"
 	icon = 'icons/mecha/mecha.dmi'
+	var/emissive_appearance_icon = 'icons/mecha/mecha_emissive.dmi'
```

Comment:
Could you move this down a bit with the other vars that are being declared and autodoc it? Even if it is just a small explanation (not a lot of people know about emissives).

Reply:
Has been addressed.